backup_pom_test_config

```diff
@@ -0,0 +1,101 @@
+<dependency>
```

Comment:
A to co za śmieć? :) Masz gita od backupowania. Zawsze możesz cofnąć się do dowolnego commita. Jakbyś zgubił commita, to jest jeszcze git reflow. A na samym końcu jest jeszcze local history w Idei.

Reply:
+